credential/reclaim/v0.1.0/schema.json

```diff
@@ -0,0 +1,133 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://common.schemas.verida.io/credential/reclaim/v0.1.0/schema.json",
+    "title": "Schema for Reclaim Protocol",
+    "description": "A schema for reclaim protocol credential",
+    "type": "object",
+    "appearance": {
+        "style": {
+            "color": "#4bbf95",
+            "icon": "./icon.svg"
+        }
+    },
+    "allOf": [
+        {
+            "$ref": "https://core.schemas.verida.io/base/latest/schema.json"
+        },
+        {
+            "properties": {
+                "appId": {
+                    "title": "App ID",
+                    "description": "Unique id of the task allocated by the allocator node",
+                    "type": "string"
+                },
+                "httpProviderId": {
+                    "title": "Http Provider Id",
+                    "description": "ID of http provider",
+                    "type": "array",
+                    "items": {
+                        "type": "string"
+                    }
+                },
+                "proofs": {
+                    "title": "Session Id",
+                    "description": "Values of public fields defined in schema",
+                    "type": "array",
+                    "items": {
```

Comment:
You can also add a `required` under `items` to list the mandatory properties

Reply:
If proofs shall not be empty, you can also specify `"minItems: 1` or whatever number.

Reply:
Done